### ISSUE TITLE
Keep board statistics

### ISSUE THREAD

**Issue by rowleya** [SpiNNakerManchester/spalloc_server#66]
Keep track of how often individual boards are used, either by themselves or as part of a bigger machine.  This will help to diagnose errors that occur in running of jobs.

**Comment by Christian-B** [comment]
Even cooler would be if these stats also reported if the job ran successfully or what error caused them to die.

**Comment by rowleya** [comment]
Yes, if the executor of the job could report errors that would also help, and could be stored alongside these statistics.
